front-end/src/components/RegistrationEventPage.vue

```diff
@@ -0,0 +1,945 @@
+    },
+    {
+      question:
+        "Existe alguma estratégia no evento para reduzir o desperdício de alimentos e materiais descartáveis?",
+      answer: form.value.reduceWaste,
+    },
+    {
+      question:
+        "É possível chegar facilmente ao local do evento por meio de transporte público?",
+      answer: form.value.publicTransportation,
+    },
+    {
+      question:
+        "Há incentivos para que os participantes utilizem meios de transporte sustentáveis?",
+      answer: form.value.sustainableTransportIncentive,
+    },
+    {
+      question: "O evento beneficia alguma causa social ou ambiental?",
+      answer: form.value.socialEnvironmental,
+    },
+    {
+      question: "O evento monitora e avalia seu impacto ambiental?",
+      answer: form.value.monitorImpact,
+    },
+  ];
+
+  const sustainabilityQuestions = questionsAndAnswers.filter(
+    (qa) => qa.answer !== undefined
+  );
+
+  if (sustainabilityQuestions.length > 0) {
+    requestBody.questions = sustainabilityQuestions;
+  }
+
+  delete requestBody.renewableEnergy;
+  delete requestBody.wasteManagement;
+  delete requestBody.visibleBins;
+  delete requestBody.reduceWaste;
+  delete requestBody.publicTransportation;
+  delete requestBody.sustainableTransportIncentive;
+  delete requestBody.socialEnvironmental;
+  delete requestBody.monitorImpact;
+
+  return requestBody;
+}
+
+function changeCity(event: Event) {
+  const selectElement = event.target as HTMLSelectElement;
+  form.value.cityId = Number(selectElement.value);
+}
+
+function changeLocation(event: Event) {
+  const selectElement = event.target as HTMLSelectElement;
+  form.value.locationId = Number(selectElement.value);
+}
+
+function selectYes() {
+  isYesSelected.value = true;
+  form.value.initialPrice = 0;
+  form.value.finalPrice = 0;
+}
+
+function selectNo() {
+  isYesSelected.value = false;
+}
+
+function showLocalFields() {
+  console.log("showLocalFields() foi chamado");
+  showLocalSelect.value = !showLocalSelect.value;
+
+  if (!showLocalSelect.value) {
+    requiredFields.push("locationId");
+    requiredFields.splice(requiredFields.indexOf("address"), 1);
+    requiredFields.splice(requiredFields.indexOf("zipcode"), 1);
+    requiredFields.splice(requiredFields.indexOf("maxCapacity"), 1);
+  } else {
+    requiredFields.push("address");
+    requiredFields.push("zipcode");
+    requiredFields.push("maxCapacity");
+    requiredFields.splice(requiredFields.indexOf("locationId"), 1);
+  }
+}
+
+onMounted(async () => {
+  cities.value = await axios
+    .get("http://localhost:3000/cities")
+    .then((res) => res.data);
+
+  locations.value = await axios
+    .get("http://localhost:3000/locations")
+    .then((res) => res.data);
+
+  categories.value = await axios
+    .get("http://localhost:3000/categories")
+    .then((res) => res.data);
+
+  categoriesIds.value = categories.value.map((category) => category.id);
+});
+</script>
+<style scoped>
+body {
+  color: black;
```

Comment:
Por que essa troca de cor para black é necessária? 

Se for realmente necessária, trocar o color: black pra classe container desse componente, mudar as coisas do body mesmo que em um contexto que o CSS ta scoped não é legal, alterações no css do body devem estar no css geral

Reply:
Color black removida sem qualquer mudança no código e alteração na pagina